### PR TITLE
Add initial documentation and notes on the idfint deployment

### PR DESCRIPTION
**Pull request by spenczar**
Added a connection guide and some examples.

Wasn't totally sure how to modify the top-level README and keep things manageable so I just made a new docs/ folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Note that in operations, users will query the PPDB using [ADQL](http://www.ivoa.
 ### Software
 
 Libraries for parsing Avro files are widely available.
-In Python, the [fastavro](https://github.com/fastavro/fastavro/) package is a good choice.
```
Comment (spenczar) [inline README.md:42]: Sorry for all the whitespace noise, grrr.

File: doc/alert_stream_integration_endpoint.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+The Kafka cluster's bootstrap URL is **alert-stream-int.lsst.cloud:9094**.
+
+TLS (aka SSL) and SCRAM-SHA-512 are required to connect; the broker is using LetsEncrypt as the CA for its certificates.
+
+In other words, the broker is using `security.protocol=SASL_SSL` and `sasl.mechanisms=SCRAM-SHA-512`.
+
+### Schema Registry
+
+The Schema Registry is available at **https://alert-scheams-int.lsst.cloud**.
```
Comment (ebellm) [inline doc/alert_stream_integration_endpoint.md]:
```suggestion
The Schema Registry is available at **https://alert-schemas-int.lsst.cloud**.
```

File: doc/alert_stream_integration_endpoint.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+All GET requests are permitted without authentication, while all non-GET requests will be denied.
+
+### Topics
+
+The only topic that is accessible is `alerts-simulated`.
+This is the topic that receives a complete visit of sample alert data every 37 seconds.
+
+Community brokers have `Read` and `DescribeConfigs` permissions for this topic; see [the Kafka documentation](https://kafka.apache.org/documentation/#operations_resources_and_protocols) for a precises explanation of what this covers.
```
Comment (ebellm) [inline doc/alert_stream_integration_endpoint.md]:
```suggestion
Community brokers have `Read` and `DescribeConfigs` permissions for this topic; see [the Kafka documentation](https://kafka.apache.org/documentation/#operations_resources_and_protocols) for a precise explanation of what this covers.
```

File: doc/alert_stream_integration_endpoint.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+### Consumer Groups
+
+Community brokers are granted global permissions of consumer groups that are prefixed with their username.
```
Comment (ebellm) [inline doc/alert_stream_integration_endpoint.md]:
```suggestion
Community brokers are granted global permissions on consumer groups that are prefixed with their username.
```

**Review by ebellm — APPROVED** [review]
Minor typos only.

**Pull request merged**